dot_calling.ipynb

```diff
@@ -0,0 +1,2181 @@
+{
```

Comment:
<p>finds peaks in contact frequency that are enriched above a local background. </p><p> </p><p>I'd mention the overall set of steps used to call the dots, and define the idea of a kernel etc, before saying that this set of steps was originally implemented as Hiccups</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>